invenio_communities/assets/semantic-ui/js/invenio_communities/settings/privileges.js

```diff
@@ -117,8 +127,10 @@ class CommunityPrivilegesForm extends Component {
                   icon
                   labelPosition="left"
                   loading={isSubmitting}
+                  toggle
+                  active={isSaved}
                 >
-                  <Icon name="save"></Icon>Save
+                  <Icon name="save"></Icon>{isSaved ?'Saved': 'Save'}
```

Comment:
Maybe the `Saved` text should be shown for a limited amount of time and then go back to `Save` ? I am thinking about the case that the user saves, then button becomes Saved but then they want to change the value again so they will need to click on the `Saved` button for saving again....I think it feels a bit weird from UX perspective...

Reply:
but this will change to save as soon as they change a value.